sdk/src/state.rs

```diff
@@ -453,7 +453,7 @@ impl State {
             .cloned()
             .map(|code| CodeResponse {
                 creator: code.creator.into(),
-                hash: sha256(&code.wasm_byte_code).into(),
+                hash: sha256(&*code.wasm_byte_code).into(),
```

Comment:
i kinda dislike `&*`. it looks confusing to me why you would deref (`*`) then immediately ref (`&`) again. `as_slice` is more straightforward although it does exactly the same thing as `&*` here

```suggestion
                hash: sha256(code.wasm_byte_code.as_slice()).into(),
```